devDocs/startupShutdownInstall.md

```diff
@@ -0,0 +1,96 @@
+# NVDA Starting and Exiting outline
+
+## As a user, NVDA can be started by:
+
+1. For an installed copy:
+    1. Ctrl+Alt+N
+    1. Via secure desktop/ease of access
+    1. Via startup
+1. For an installed copy, portable copy, installer:
+    - By running the exe (see cli options too).
+    This can be triggered by a user or external process such as an existing NVDA instance
+    - test: `startupShutdownNVDA.Starts`
+1. For source: eg runnvda.bat
+
+## As a user, NVDA can be shutdown by:

```

Comment:
I think the "as a user" part can be removed from here and from the previous heading. Some of these aren't direct user actions and I'm not sure it is a useful distinction.

I suppose there are two "views" of shutdown / startup that are useful:
- What can I "a tester" do to exercise all the startup / shutdown pathways?
  - Normal shutdown
     - Trigger via one of (NVDA+q, NVDA+n and select exit)
     - System tests
        - "startupShutdownNVDA.Quits from menu"
  - Etc
- What do I "a developer" need to be aware of for startup / shutdown mechanisms"
   -  Normal shutdown (most common):
      - UI calls `triggerNVDAExit`
      - All wx windows close (Note, "NVDA window class" no longer present)
      - wx event loop ends
      - etc
   - receive WM_QUIT:
      - wx Mainloop exits
      - triggerNVDAExit is called
   - restarting due to UI
      - core.restart called, this calls core.triggerNVDAExit
   - restarting due to crash
      - core.restartUnsafely called 
      - Nothing can be done about current instance, just start a new instance.

Reply:
```suggestion
## NVDA can be shutdown by:
```

Reply:
Yep, these were the two views I had. Just phrasing the titles was awkward.